### ISSUE TITLE
Push notification is not coming in IOS devices but working fine in android device.

### ISSUE THREAD

**Issue by jayantjha23** [firebase/firebase-unity-sdk#936]
### [REQUIRED] Please fill in the following fields:

  * Unity editor version: ____2021.3.28f_
  * Firebase Unity SDK version: ____11.6.0_
  * Source you installed the SDK: _.unitypackage____ (.unitypackage or Unity Package Manager)
  * Problematic Firebase Component:    messaging_____ (Auth, Database, etc.)
  * Other Firebase Components in use: __Auth,App,Crashlytics,Analytics___ (Auth, Database, etc.)
  * Additional SDKs you are using: __Flurry,Ironsource sdk.___ (Facebook, AdMob, etc.)
  * Platform you are using the Unity editor on:    Mac _____ (Mac, Windows, or Linux)
  * Platform you are targeting: ___iOS,Android__ (iOS, Android, and/or desktop)
  * Scripting Runtime: __IL2CPP___ (Mono, and/or IL2CPP)
  * Pre-built SDK from the [website](https://firebase.google.com/download/unity) or open-source from this repo: _____

**Comment by paulinon** [comment]
Hi @jayantjha23,

Could you confirm if you uploaded an APNs certificate/auth key as well as enabled Push Notifications inside Capabilities? Take note that there are additional steps in [setting up an FCM app](https://firebase.google.com/docs/cloud-messaging/unity/client#enable_push_notifications_on_apple_platforms) for an iOS build. You may provide the detailed steps you've taken before facing the issue so we can identify what's causing this to happen.

Additionally, does the issue persist when using our [quickstart project](https://github.com/firebase/quickstart-unity/tree/master/messaging/testapp)?

**Comment by jayantjha23** [comment]
Yes  I have uploaded APNs and it is working fine with Firebase Unity SDK version 9.6.0.

**Comment by jayantunity** [comment]
any update?
